src/Database/AppProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace BeyondCode\LaravelWebSockets\Database;
+
+use BeyondCode\LaravelWebSockets\Apps\App;
+use BeyondCode\LaravelWebSockets\Apps\AppProvider as IAppProvider;
+use BeyondCode\LaravelWebSockets\Database\Models\App as DatabaseApp;
+
+class AppProvider implements IAppProvider
+{
+    public function all(): array
+    {
+        return DatabaseApp::where('active', true)->get()->map(function (DatabaseApp $app) {
+            return $this->instantiate($app->toArray());
+        })->toArray();
+    }
+
+    public function findById($appId): ?App
+    {
+        return $this->instantiate(DatabaseApp::find($appId)->toArray() ?? null);
+    }
+
+    public function findByKey(string $appKey): ?App
+    {
+        return $this->instantiate(DatabaseApp::where('key', $appKey)->first()->toArray() ?? null);
```

Comment:
@masterix21 If you submit an unknown appKey to this function the websocket-service will crash.
in this cas `DatabaseApp::where('key', $appKey)->first()` is null and `null->toArray()` throws an exception